src/main/java/az/edu/turing/mergeSortedArrays/MergeSortedArrays.java

```diff
@@ -0,0 +1,29 @@
+package az.edu.turing.mergeSortedArrays;
+
+
+public class MergeSortedArrays {
+    public static void main(String[] args) {
+        int[] nums1 = {4, 5, 6, 0, 0, 0};
+        int[] nums2 = {5, 7, 8};
+        mergeSortedArrays(nums1, 3, nums2, 3);
+        for (int num : nums1) {
+            System.out.print(num + " ");
+        }
+    }
+
+    public static void mergeSortedArrays(int[] nums1, int m, int[] nums2, int n) {
```

Comment:
do we need send sizes as parameter in method? we can get this information by nums.lenght